src/build.ts

```diff
@@ -7,9 +7,9 @@ import decompress from "decompress";
 import shelljs from "shelljs";
 import tar from "tar";
 // src
-import { getDirname, getVersion } from "./module-specific.js";
+import { getDirname } from "./module-specific.js";
 
-const version = getVersion();
+const version = "0.0.1-alpha-3";
```

Comment:
Not sure this is the slickest way to specify this, but it works. Open to other ideas.